.github/workflows/jira-issue-create.yml

```diff
@@ -4,6 +4,13 @@ name: Jira Issue Creator
 on:
   issues:
     types: [opened]
+  workflow_call:
+    inputs:
+      label:
+        required: true
+        type: string
```

Comment:
Curious how other repos can use this action then? And how ts can trigger this action?

Reply:
https://github.com/yuhao900914/Amplitude-Flutter-Example/issues
This is the GitHub action of my example app. 
Other repos need to call this template with the required input and label and secret. 
I need to update the Jira create GitHub action in other repos as well.

Defined the specific secrets works fine. But I'm still trying to make it use inherit secrets to avoid defining multiple secrets in other repo.

Reply:
I saw the secrets in this repo Settings. As we don't have a mid-level group between `amplitude` and `Amplitude-TypeScript`, curious how does the inherit work.

Reply:
amplitude is the organization. Amplitude-typescript is the repo. 
I think it should be if other repos like Amplitude-Javascript call this GitHub template without any secrets settings, it will use the secrets under this repo. 
https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/reusing-workflows

`Workflows that call reusable workflows in the same organization or enterprise can use the inherit keyword to implicitly pass the secrets.`